core/chains/evm/txmgr/broadcaster_test.go

```diff
@@ -161,6 +161,7 @@ func TestEthBroadcaster_LoadNextSequenceMapFailure_StartupSuccess(t *testing.T)
 	// Instance starts without error even if loading next sequence map fails
 	err := eb.Start(testutils.Context(t))
 	require.NoError(t, err)
+	t.Cleanup(func() { assert.NoError(t, eb.Close()) })
```

Comment:
Seems like an unreachable code in case `err != nil` on the previous line
`require` aborts test execution https://github.com/stretchr/testify/blob/v1.8.4/require/require.go#L1360

Reply:
Yes, but if we fail to start then we don't need to call close. It is up to the Start() method to clean up after itself in that case.

Reply:
@jmank88 ah i see, i guess i was too eager to comment:) Thanks and resolved!